app/components/elements/TextLink.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box } from 'rebass/styled-components';
+import { TextLink as Text } from '../elements/FontStyles';
+
+export const TextLink = props => {
+  const { linkText, link, ...textLinkProps } = props;
+
+  return (
+    <div>
```

Comment:
We shouldn't be wrapping the element in 2 divs.  This prevents the link from being used inline.